packages/lodestar/test/spec/sanity/blocks/sanity_blocks_mainnet.test.ts

```diff
@@ -5,7 +5,7 @@ import {equals} from "@chainsafe/ssz";
 
 import {BeaconBlock, BeaconState} from "@chainsafe/eth2.0-types";
 import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
-import {stateTransition} from "../../../../src/chain/stateTransition";
+import {stateTransition} from "../../../../../eth2.0-state-transition/src";
```

Comment:
@ec2 should be "@chainsafe/eth2.0-state-transition" everywhere :)

Reply:
oops missed that one. but checkout blockAssembly.test.ts.

Reply:
actually im sure i missed all of them except blockAssembly.test.ts

Reply:
Nvm, i havent tested the spec tests yet. Could you look at the lodestar unit test?